javascript/index.js

```diff
@@ -14,52 +13,93 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  setInterval(() => {
+    printMinutes();
+    printSeconds();
+  }, 100)
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+  minUni.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secDec.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[0];
+  secUni.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[1];
 }
 
+
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+
+  
 }
 
 function printSplit() {
   // ... your code goes here
+  splits.innerHTML += `<div>${chronometer.splitClick()}</div>`;
 }
 
 function clearSplits() {
-  // ... your code goes here
+
+  splits.innerHTML = '';
+
 }
 
 function setStopBtn() {
-  // ... your code goes here
+
+  chronometer.setStopBtn();
+
+
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  chronometer.setSplitBtn();
```

Comment:
Same thing is happening here as above, `chronometer.setSplitBtn()` will not return anything. Use the logic you've applied in the event listener here as well and call this `setSplitBtn()` method in the event listener instead

```suggestion
  btnRight.className = 'btn split';
  btnRight.innerText = 'SPLIT';
```